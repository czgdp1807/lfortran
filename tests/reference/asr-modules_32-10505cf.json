{
    "basename": "asr-modules_32-10505cf",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/modules_32.f90",
    "infile_hash": "f257d943c6ae6e45cd8d6612a542f924bcc97e2c9033adfe224bf780",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-modules_32-10505cf.stdout",
    "stdout_hash": "9ded680bdc4ab8405f85a8c33208659f1dd39c7b4d4fef769c1e6b0c",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}