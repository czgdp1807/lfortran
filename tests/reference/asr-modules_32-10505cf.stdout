(TranslationUnit
    (SymbolTable
        1
        {
            fpm_targets:
                (Module
                    (SymbolTable
                        2
                        {
                            build_target_ptr:
                                (StructType
                                    (SymbolTable
                                        3
                                        {
                                            ptr:
                                                (Variable
                                                    3
                                                    ptr
                                                    []
                                                    Local
                                                    (PointerNullConstant
                                                        (Pointer
                                                            (Struct
                                                                2 build_target_t
                                                                []
                                                            )
                                                        )
                                                    )
                                                    ()
                                                    Default
                                                    (Pointer
                                                        (Struct
                                                            2 build_target_t
                                                            []
                                                        )
                                                    )
                                                    Source
                                                    Public
                                                    Required
                                                    .false.
                                                )
                                        })
                                    build_target_ptr
                                    [build_target_t]
                                    [ptr]
                                    Source
                                    Public
                                    .false.
                                    ()
                                    ()
                                ),
                            build_target_t:
                                (StructType
                                    (SymbolTable
                                        4
                                        {
                                            output_file:
                                                (Variable
                                                    4
                                                    output_file
                                                    []
                                                    Local
                                                    ()
                                                    ()
                                                    Allocatable
                                                    (Character 1 -2 () [])
                                                    Source
                                                    Public
                                                    Required
                                                    .false.
                                                ),
                                            version:
                                                (Variable
                                                    4
                                                    version
                                                    []
                                                    Local
                                                    ()
                                                    ()
                                                    Allocatable
                                                    (Character 1 -2 () [])
                                                    Source
                                                    Public
                                                    Required
                                                    .false.
                                                )
                                        })
                                    build_target_t
                                    []
                                    [output_file
                                    version]
                                    Source
                                    Public
                                    .false.
                                    ()
                                    ()
                                )
                        })
                    fpm_targets
                    []
                    .false.
                    .false.
                ),
            modules_32:
                (Program
                    (SymbolTable
                        5
                        {
                            
                        })
                    modules_32
                    []
                    [(Print
                        ()
                        [(StringConstant
                            "running modules_32 program"
                            (Character 1 26 () [])
                        )]
                        ()
                        ()
                    )]
                )
        })
    []
)
